var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -751,6 +858,30 @@ def cmake_args(self):
 
         return options
 
+    def setup_environment(self, spack_env, run_env):
+        if '+cuda' in self.spec:
+            if not self.stage.source_path:
+                self.stage.fetch()
+                self.stage.expand_archive()
+            # The nvcc_wrapper is also needed by client software, so save this
+            # path to be used in the function 'install_additional_files'.  We
+            # save this path becuase self.stage.source_path points to the wrong
+            # location when called from 'install_additional_files.'
+            self.nvcc_wrapper = join_path(self.stage.source_path,
+                                          'packages', 'kokkos', 'bin',
+                                          'nvcc_wrapper')
+            spack_env.set('OMPI_CXX', self.nvcc_wrapper)
+            spack_env.set('NVCC_WRAPPER_DEFAULT_COMPILER',
+                          '{0}'.format(env['CXX']))
+
+    @run_after('install')
```

Comment:
Is this step necessary?  I'm using 12.16, and nvcc_wrapper already gets placed in the bin directory.